taggui/models/image_list_model.py

```diff
@@ -136,7 +136,7 @@ def load_directory(self, directory_path: Path):
                             if any(value in orientations
                                    for value in (5, 6, 7, 8)):
                                 dimensions = (dimensions[1], dimensions[0])
-                    except (KeyError, NoParser) as exception:
+                    except Exception as exception: # (KeyError, NoParser, IndexError)
```

Comment:
I agree with this change, because the only thing that can go wrong here is reading the Exif tags, and it seems to have many types of failure cases.